StockTrader/src/main/java/com/restResource/StockTrader/entity/CompositeKey.java

```diff
@@ -0,0 +1,8 @@
+package com.restResource.StockTrader.entity;
+
+import java.io.Serializable;
+
+public class CompositeKey implements Serializable {
```

Comment:
CompositeKey is too generic of a name. Also you don't seem to be using it. Buy/SellTrigger should replace the userId and stockSymbol fields with this class, and then it should be second param in CrudRepository